src/plugins/intel_gpu/src/kernel_selector/cl_kernels/reorder_weights_int4.cl

```diff
@@ -5,7 +5,60 @@
 #include "include/batch_headers/fetch_weights.cl"
 
 KERNEL(reorder_weights_int4)(const __global INPUT0_TYPE* input, __global OUTPUT_TYPE* output) {
-#if defined(INPUT0_LAYOUT_IOYX) && defined(OUTPUT_LAYOUT_OIYX)
+#if defined(INPUT0_LAYOUT_OIYX) && defined(OUTPUT_LAYOUT_OIYX)
```

Comment:
This kernel works only when `INPUT_INNERMOST_PITCH` is an odd number, and `OUTPUT_INNERMOST_PITCH` is an even number. Could you clarify that in the source code with additional #error?

Reply:
Add condition to check even number for output innermost num.